storage/innobase/xtrabackup/src/keyring_components.cc

```diff
@@ -174,7 +160,9 @@ bool keyring_init_online(MYSQL *connection) {
       init_components = true;
       component_name = row[1];
       component_urn += row[1];
-      create_component_config_data(mysql_result);
+      if (!check_componenet_config_file()) {
+        xb::error() << "Error reading " << component_name << " config file";
+      }
```

Comment:
Should we abort here?

Reply:
I would leave this to when decrypting some data is necessary. There is a case where one might have the component configured but no tables encrypted yet.

Reply:
ok, sometimes, it is hard to catch those missing keyring config errors. (IO errors etc). Lets leave it as is